tutorials/R/api-token/api-token.Rmd

```diff
@@ -0,0 +1,266 @@
+the bottom of the My Account page, you should see this bar: 
+
+<figure>
+	<a href="{{ site.baseurl }}/images/NEON-api-token/get-api-token-button.png">
+	<img src="{{ site.baseurl }}/images/NEON-api-token/get-api-token-button.png"></a>
+	<figcaption>Screenshot of account page button to Get API Token. 
+	Source: National Ecological Observatory Network (NEON)
+	</figcaption>
+</figure>
+
+Click the 'GET API TOKEN' button. After a moment, you should see this:
+
+<figure>
+	<a href="{{ site.baseurl }}/images/NEON-api-token/api-token-view.png">
+	<img src="{{ site.baseurl }}/images/NEON-api-token/api-token-view.png"></a>
+	<figcaption>Screenshot of account page with API token created. 
+	Source: National Ecological Observatory Network (NEON)
+	</figcaption>
+</figure>
+
+Click on the Copy button to copy your API token to the clipboard:
+
+<figure>
+	<a href="{{ site.baseurl }}/images/NEON-api-token/api-token-copy-button.png">
+	<img src="{{ site.baseurl }}/images/NEON-api-token/api-token-copy-button.png"></a>
+	<figcaption>Screenshot of account page with API token created. 
+	Source: National Ecological Observatory Network (NEON)
+	</figcaption>
+</figure>
+
+## Use API token in neonUtilities 
+
+In the next section, we'll walk through saving your token somewhere secure but 
+accessible to your code. But first let's try out using the token the easy way.
+
+First, we need to load the `neonUtilities` package:
+
+```{r loadStuff, eval=FALSE, comment=NA}
+
+# install neonUtilities - can skip if already installed
+install.packages("neonUtilities")
+
+# load neonUtilities
+library(neonUtilities)
+
+```
+
+NEON API tokens are very long, so it would be annoying to keep pasting the 
+entire text string into functions. Assign your token an object name:
+
+```{r nameToken, eval=FALSE, comment=NA}
+
+NEON_TOKEN <- "PASTE YOUR TOKEN HERE"
+
+```
+
+Now we'll use the `loadByProduct()` function to download data. Your 
+API token is entered as the optional `token` input parameter. For 
+this example, we'll download Plant foliar traits (DP1.10026.001).
+
+```{r getCFC, eval=FALSE, comment=NA}
+
+foliar <- loadByProduct(dpID="DP1.10026.001", site="all", 
+                        package="expanded", check.size=F,
+                        token=NEON_TOKEN)
+
+```
+
+You should now have data saved in the `foliar` object; the API 
+silently used your token. If you've downloaded data without a 
+token before, you may notice this is faster!
+
+This format applies to all `neonUtilities` functions that involve 
+downloading data or otherwise accessing the API; you can use the 
+`token` input with all of them. For example, when downloading 
+remote sensing data:
+
+```{r getAOP, eval=FALSE, comment=NA}
+
+chm <- byTileAOP(dpID="DP3.30015.001", site="WREF", 
+                 year=2017, check.size=F,
+                 easting=c(571000,578000), 
+                 northing=c(5079000,5080000), 
+                 savepath="/data",
```

Comment:
Hi @cklunch @mjones01 sorry for the delay, I have been barely keeping up with WFH projects for the last couple of days but I've finally caught my breath! 

The new working directory format that I would like to transition to is to 1) save the working directory path as a string called 'wd' and 2) to use paste0() to concatenate wd with any file names or additional pathways as below:

`wd <- "~/Git/data/". # This will depend on your local environment`
`setwd(wd)`
...
`r <- raster(paste0(wd,"SJER_easting_northing.tif"))`

That said, I can see that these code chunks are eval=F, and therefore it doesn't matter _as much_. I think that it would still be good form to match this convention, and I will be happy to edit these files directly through the GitHub interface to harmonize with the other tutorials if y'all agree.

Reply:
@donal-at-NEON  I've updated the tutorial to set a working directory and use it for download/storage. Thanks!